servers/src/main/java/tachyon/master/next/filesystem/meta/InodeDirectory.java

```diff
@@ -34,16 +34,19 @@
 public final class InodeDirectory extends Inode {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
-  private IndexedSet.FieldIndex mIdIndex = new IndexedSet.FieldIndex<Inode>() {
+  private IndexedSet.FieldIndex<Inode> mIdIndex = new IndexedSet.FieldIndex<Inode>() {
+    @Override
     public Object getFieldValue(Inode o) {
       return o.getId();
     }
   };
-  private IndexedSet.FieldIndex mNameIndex = new IndexedSet.FieldIndex<Inode>() {
+  private IndexedSet.FieldIndex<Inode> mNameIndex = new IndexedSet.FieldIndex<Inode>() {
+    @Override
     public Object getFieldValue(Inode o) {
       return o.getName();
     }
   };
+  @SuppressWarnings("unchecked")
```

Comment:
To suppress the warning: A generic array of IndexedSet.FieldIndex<Inode> is created for 
 a varargs parameter